components/ClubPage/Header.js

```diff
@@ -27,7 +27,7 @@ const Header = ({ club, userInfo, favorites, style, updateFavorites }) => {
     (inClub && inClub.role <= ROLE_OFFICER) ||
     (userInfo && userInfo.is_superuser)
 
-  const [favCount, setFavCount] = useState(club.favorite_count || 0)
+  const [favCount] = useState(club.favorite_count || 0)
```

Comment:
If not using the setter, why not just have
```suggestion
  const favCount = club.favorite_count || 0
```

Reply:
I think this lint warning is being handled in another PR

Reply:
I believe that #61 might use the setter, we can handle this during the merge process.